linux_service/moneo_prestart.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+MONEO_PATH=$1
+EXE_TYPE=$2
+
+# check MONEO_PATH variable is set if not set to default
+if [[ -z "$MONEO_PATH" ]];
+then
+    MONEO_PATH=/opt/azurehpc/tools/Moneo
+    echo 'default Moneo path used'
+fi
+echo "Moneo path=$MONEO_PATH"
+
+if [[ ! -z "$EXE_TYPE" ]];
```

Comment:
Is if else needed or can this be:

```shell
if [[ -z "$EXE_TYPE" ]];
then
    echo 'No executable passed in'
    exit 1
fi
```

Reply:
I want the script to fail if the argument is not passed but maybe I can make the message clear that we are going to exit

Reply:
I now understand the comment disregard last

Reply:
addressed